transaction/tests/LockTable_unittest.cpp

```diff
@@ -43,62 +43,62 @@ class LockTableTest : public ::testing::Test {
+            LockTableResult::kAlreadyInOwned);
 
   // Another transaction acquires compatible lock on the same resource.
   EXPECT_EQ(lock_table_.putLock(tid_2_,
                                 ResourceId(2),
-                                AccessMode(AccessModeType::kSLock)),
-            LockTableResult::kPLACED_IN_OWNED);
+                                AccessMode::SLockMode()),
+            LockTableResult::kPlacedInOwned);
 }
 
 TEST_F(LockTableTest, IncompatibleRequestsFromDifferentTransactions) {
   EXPECT_EQ(lock_table_.putLock(tid_1_,
                                 ResourceId(2),
-                                AccessMode(AccessModeType::kIsLock)),
-            LockTableResult::kPLACED_IN_OWNED);
+                                AccessMode::IsLockMode()),
+            LockTableResult::kPlacedInOwned);
 
   // Acquire the same lock mode on same resource.
   EXPECT_EQ(lock_table_.putLock(tid_1_,
                                 ResourceId(2),
-                                AccessMode(AccessModeType::kIsLock)),
-            LockTableResult::kALREADY_IN_OWNED);
+                                AccessMode::IsLockMode()),
+            LockTableResult::kAlreadyInOwned);
 
   // Another transaction acquires incompatible lock on the same resource.
   EXPECT_EQ(lock_table_.putLock(tid_2_,
                                 ResourceId(2),
-                                AccessMode(AccessModeType::kXLock)),
-            LockTableResult::kPLACED_IN_PENDING);
+                                AccessMode::XLockMode()),
+            LockTableResult::kPlacedInPending);
 }
 
 TEST_F(LockTableTest, StarvationProtection) {
   EXPECT_EQ(lock_table_.putLock(tid_1_,
                                 ResourceId(2),
-                                AccessMode(AccessModeType::kIsLock)),
-            LockTableResult::kPLACED_IN_OWNED);
+                                AccessMode::IsLockMode()),
+            LockTableResult::kPlacedInOwned);
 
   // Another transaction requests incompatible lock on the same resource.
   // It should wait for the previous transaction.
   EXPECT_EQ(lock_table_.putLock(tid_2_,
                                 ResourceId(2),
-                                AccessMode(AccessModeType::kXLock)),
-            LockTableResult::kPLACED_IN_PENDING);
+                                AccessMode::XLockMode()),
+            LockTableResult::kPlacedInPending);
 
   // Another third transaction requests a compatible lock on the same resource.
   // Normally, it should acquire the lock, however, there is a pending
   // transaction waiting on the same resource. To prevent starvation, we should
   // put in the pending list.
   EXPECT_EQ(lock_table_.putLock(tid_3_,
                                 ResourceId(2),
-                                AccessMode(AccessModeType::kIsLock)),
-            LockTableResult::kPLACED_IN_PENDING);
+                                AccessMode::IsLockMode()),
+            LockTableResult::kPlacedInPending);
```

Comment:
FYI, `EXPECT_EQ` has a convention that the left hand side is the expected value, and the right hand side is the checked value. So, in this case, we should refactor to `EXPECT_EQ(LockTableResult::kPlacedInPending, ...);`

Please do such reverses in this file.

Also, I suggest to declare `const AccessMode is_lock(AccessModeType::kIsLock), x_lock(AccessModeType::kXLock);`, and then use the variables in `putLock`.